torch/autograd/graph.py

```diff
@@ -227,8 +227,17 @@ def increment_version(tensor: torch.Tensor) -> None:
 
     Note that incrementing the version counter multiple times for a single inplace operation
     is not problematic.
+
+    Note that if you pass in tensor constructed under torch.inference_mode(),
+    we will not bump its version counter (because your tensor does not have one).
+
+    Can accept either a tensor, or a list of tensors.
+    If you pass in a list of tensors, we will separately
     """
-    torch._C._increment_version(tensor)
+    if isinstance(tensor, list):
+        torch._C._increment_versions(tensor)
+    else:
+        torch._C._increment_version(tensor)
```

Comment:
can we have only one binding for the sequence version and just wrap single Tensor into a one element tuple here?

Reply:
sgtm!